### DISCUSSION TITLE
JSON Extension doesn’t package properly

### DISCUSSION THREAD

**Discussion by kindpump** [TurboWarp/extensions#309] [Q&A] [answered]
I'm working on a game and the JSON extension in the turbowarp gallery is absolutely amazing. I implemented it into a ton of elements of my game and when I package the project, (I tried outputting as HTML as well as an electron program) it doesn't function at all, even though the other extensions I have in the game do. Any ideas on how to fix this?

**Comment by GarboMuffin** [comment]
Elaborate on what "it doesn't function at all" means. Error messages? Stuck progress bars? Something else?

If you're using the desktop app, you need the beta: https://github.com/TurboWarp/desktop/releases/tag/v1.7.2-beta.1

**Reply by kindpump** [reply]
So when the project is packaged, instead of functioning (getting JSON values, setting lists etc), the blocks output an empty value no matter what I do

**Reply by Skyhigh173** [reply]
That's weird

**Comment by MA-master-sec** [comment, marked as answer]
However,I can use it.

you could try to use text instead?

**Reply by Skyhigh173** [reply]
what do you mean by 'text'? 'text extension'?

**Reply by MA-master-sec** [reply]
y

**Comment by Supersonic734** [comment]
I'm having the same issue, and can't figure it out.

**Reply by GarboMuffin** [reply]
Email the broken packaged project to contact@turbowarp.org

**Reply by Brackets-Coder** [reply]
I'd be happy to help where I can as well
